benchmark/config.h

```diff
@@ -23,14 +23,14 @@ class Config {
   Config(const Config& other) = delete;
   Config& operator=(const Config& other) = delete;
 
-  const std::vector<Vendor>& vendors() const { return vendors_; }
+  const std::set<Vendor>& vendors() const { return vendors_; }
   bool check_results() const { return check_results_; }
 
   bool Parse(int argc, char** argv, const Options& options);
 
  protected:
   base::FlagParser parser_;
-  std::vector<Vendor> vendors_;
+  std::set<Vendor> vendors_;
```

Comment:
Is there a reason you didn't change the `vendors_` member in `SimpleReporter`?

Reply:
Changing the type significantly impacts the code in `SimpleReporter`, especially the `SimpleReporter::Show()` method. Since `std::set<T>` lacks an `operator[]`, modifying the code to accommodate this would result in a less elegant solution.